Sources/AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -27,6 +27,8 @@
 --structthreshold 20 # organizeDeclarations
 --enumthreshold 20 # organizeDeclarations
 --organizetypes class,struct,enum,extension,actor # organizeDeclarations
+--visibilityorder beforeMarks,instanceLifecycle,open,public,package,internal,fileprivate,private # organizeDeclarations
+--typeorder nestedType,staticProperty,staticPropertyWithBody,classPropertyWithBody,instanceProperty,instancePropertyWithBody,staticMethod,classMethod,instanceMethod # organizeDeclarations
```

Comment:
@calda is it expected that there is no `classProperty`?

Reply:
There's no such thing as a stored class property (aka a class property without a body)! Found this out when I was first working on the `organizeDeclarations` rule: https://forums.swift.org/t/class-properties/16539

Reply:
Ah! Good to know.